lms/urls.py

```diff
@@ -685,15 +685,15 @@
     ),
 ]
 
-if settings.FEATURES.get('ENABLE_JOURNAL_INTEGRATION'):
-    urlpatterns += [
-        url(
-            r'^bundles/{}/about'.format(
-                r'(?P<bundle_uuid>[0-9a-f-]+)',
-            ),
-            include('openedx.features.journals.urls')
+
+urlpatterns += [
```

Comment:
why we have removed feature check condition here?

Reply:
I have to use this URL in my test cases. By default `settings.FEATURES ['ENABLE_JOURNAL_INTEGRATION']` is false. I tried to enable it by mocking, but coundn't access the URL.

Reply:
I think we should try to figure out why you are not being able to override this instead of removing the condition here.

Reply:
I agree we should try to figure out how to override rather than change this

Reply:
Looks like you use this in other places
```@mock.patch.dict(settings.FEATURES, {"ENABLE_JOURNAL_INTEGRATION": True})```
Does that not work for the url in question?